packages/@aws-cdk/aws-logs/lib/log-retention-provider/index.ts

```diff
@@ -105,6 +134,9 @@ export async function handler(event: AWSLambda.CloudFormationCustomResourceEvent
     // Parse to AWS SDK retry options
     const retryOptions = parseRetryOptions(event.ResourceProperties.SdkRetry);
 
+    //The deletion policy of log after the stack is deleted
+    const logDeletionPolicy = event.ResourceProperties.LogDeletionPolicy;
```

Comment:
You only use this value in one place so there's not really a benefit to setting it to a const.